language-server/modules/langserver-commons/src/main/java/org/ballerinalang/langserver/commons/capability/InitializationOptions.java

```diff
@@ -22,23 +22,11 @@
  */
 public interface InitializationOptions {
 
-    /**
-     * Initialization option key for pull module support.
-     */
-    String KEY_PULL_MODULE_SUPPORT = "pullModuleSupport";
```

Comment:
Any specific reason to remove these flags?

Reply:
Since we don't need any client involvement to handle pull module code action, we don't need the `initOption` named `pullModuleSupport` anymore. That's why this was removed. WDYT?

Reply:
You mean, because of the new API can handle the in memory model to do the compilation online, right? If that so, +1 to proceed.